resources/asdf-format.org/manifests/transform-1.5.0.yaml

```diff
@@ -673,6 +673,31 @@ tags:
   title: One dimensional sine model.
   description: |-
     One dimensional sine.
+- tag_uri: tag:stsci.edu:asdf/transform/cosine1d-1.0.0
+  schema_uri: http://stsci.edu/schemas/asdf/transform/cosine1d-1.0.0
```

Comment:
Since we have the capability now, any interest in creating a single `trigonometric-1.0.0` schema and sharing it among each of these?

Reply:
Sure we can do that.